ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -286,6 +286,12 @@ Item {
                     return;
                 }
 
+                if (link.startsWith('//')) {
+                  let pk = link.replace("//", "");
+                  profileClick(chatsModel.userNameOrAlias(pk), pk, chatsModel.generateIdenticon(pk))
```

Comment:
Just a thought for the future:

It'd be cool if we could have the profilepopup to load the data it needs, instead of us passing down all the needed values one by one